DOCS.md

```diff
@@ -11,8 +11,10 @@ default configuration with the following parameters:
 ```yaml
 deploy:
   anynines:
-    username: foo
-    password: bar
-    organization: drone
-    space: test
+    username: drone@drone.io
+    password: drone
+    organization: drone_drone_io
+    space: production
```

Comment:
my only comment it to use something like `octocat@github.com` and orginzation as `octocat` (for example) only because people might think they actually need to enter `drone` as the value in order to integration with drone

people are used to octocat as a demo value :)